textile/features.textile

```diff
@@ -88,9 +122,9 @@ h3(#restclient). RestClient
 *** @(RSC6b2)@ @direction@ backwards or forwards; if omitted the direction defaults to the REST API default (backwards)
 *** @(RSC6b3)@ @limit@ supports up to 1,000 items; if omitted the limit defaults to the REST API default (100)
 *** @(RSC6b4)@ @unit@ is the period for which the stats will be aggregated by, values supported are @minute@, @hour@, @day@ or @month@; if omitted the unit defaults to the REST API default (@minute@)
-* @(RSC16)@ @RestClient#time@ function sends a get request to @rest.ably.io/time@ and returns the server time in milliseconds since epoch or as a Date/Time object where suitable
+* @(RSC16)@ @RestClient#time@ function sends a get request to @main.realtime.ably.net/time@ and returns the server time in milliseconds since epoch or as a Date/Time object where suitable
```

Comment:
Surely this should reference the previously-defined `primary domain` (or RSC25) rather than hardcoding the main prod cluster endpoint